### PR TITLE
Add a new user

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds a new user.

## 💭 Motivation and context ##

The new user is now supporting PCA.

## 🧪 Testing ##

I think it's good!

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.

### CODE REVIEW

File: terraform/scripts/user_ssh_setup.yml [hunk 1]
```diff
@@ -100,6 +100,15 @@ users:
...
         AAAAC3NzaC1lZDI1NTE5AAAAILy6xA3x17WWJ1cH0BpJFeTCLqJfxEJV2N8yqO+HWvWz
         jennifer.rodrick
+  - name: mina.salehi
+    groups: [sudo]
+    sudo: ["ALL=(ALL) NOPASSWD:ALL"]
+    shell: /bin/bash
+    ssh-authorized-keys:
+      - >
+        ssh-ed25519
+        SHA256:MEw5IgyXsQtPVkjZAKWVD7i4urLANPgW7mjE3ab2Jec
```
Comment (dav3r) [inline terraform/scripts/user_ssh_setup.yml]: This is the fingerprint, not the public key, right?  Don't we need the key itself?

Comment (jsf9k) [reply terraform/scripts/user_ssh_setup.yml]: You are correct!  Fixed in commit e5c922d.

**Review by dav3r — APPROVED** [review]
Excellent! 👍

**Review by mcdonnnj — APPROVED** [review]
:shipit:

**Pull request merged**